### PR TITLE
612 revise operations form

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #612

### CODE REVIEW

File: client/app/utils/jsonSchema/operations.ts [hunk 1]
```diff
@@ -44,28 +41,46 @@ const operationPage1: RJSFSchema = {
...
+    // process_flow_diagram: {
+    //   type: "string",
+    //   title: "Process Flow Diagram",
+    //   format: "data-url",
+    // },
+    // boundary_map: {
+    //   type: "string",
+    //   title: "Boundary Map",
+    //   format: "data-url",
+    // },
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:61]: I'd recommend deleting the `process_flow_diagram` and `boundary_map` parts entirely - I don't think we'll ever be using them

File: client/app/utils/jsonSchema/operations.ts [hunk 2]
```diff
@@ -422,16 +426,21 @@ export const operationUiSchema = {
     "type",
     "naics_code_id",
     "regulated_products",
-    "reporting_activities",
+    // "reporting_activities",
+    // "process_flow_diagram",
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:430]:
```suggestion
```

File: client/app/utils/jsonSchema/operations.ts [hunk 3]
```diff
@@ -422,16 +426,21 @@ export const operationUiSchema = {
...
+    // "boundary_map",
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:431]:
```suggestion
```

File: client/app/utils/jsonSchema/operations.ts [hunk 4]
```diff
@@ -422,16 +426,21 @@ export const operationUiSchema = {
...
-    "previous_year_attributable_emissions",
-    "swrs_facility_id",
     "bcghg_id",
+    "opt_in_section",
     "opt_in",
+    "opt_in_note_section",
+    "opt_in_signed_statuatory_declaration",
     "Does the operation have multiple operators?",
     "operators",
     "percentage_ownership",
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:441]: I don't think we need lines 439-441 anymore, do we?

File: client/app/utils/jsonSchema/operations.ts [hunk 5]
```diff
@@ -476,76 +485,94 @@ export const operationUiSchema = {
...
+  point_of_contact_section: {
+    ...subheading,
+    "ui:title": PointOfContactTitle,
+  },
+  // data_flow_diagram: {
+  //   "ui:widget": "FileWidget",
+  // },
+  // boundary_map: {
+  //   "ui:widget": "FileWidget",
+  // },
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:508]: can just entirely delete lines 503-508

File: client/app/utils/jsonSchema/operations.ts [hunk 6]
```diff
@@ -446,9 +455,9 @@ export const operationUiSchema = {
...
     "phone_number",
-    "multiple_operators_section",
-    "operation_has_multiple_operators",
-    "multiple_operators_array",
+    // "multiple_operators_section",
+    // "operation_has_multiple_operators",
+    // "multiple_operators_array",
     "mo_percentage_ownership",
     "mo_mailing_address_same_as_physical",
     "mo_mailing_address_section",
```
Comment (andrea-williams) [inline client/app/utils/jsonSchema/operations.ts:463]: comment out lines 461-463?

**Review by andrea-williams — APPROVED** [review]
Some small suggestions for extra lines that can be commented out or deleted entirely. 

On behalf of @Sepehr-Sobhani and his current rebasing miseries, I'd ask that you don't merge this PR into develop until he's finished fixing his PR and got it merged, since the changes made in this PR to models.py will impact Sep's PR.

**Comment by marcellmueller** [comment]
> Some small suggestions for extra lines that can be commented out or deleted entirely.
> 
> On behalf of @Sepehr-Sobhani and his current rebasing miseries, I'd ask that you don't merge this PR into develop until he's finished fixing his PR and got it merged, since the changes made in this PR to models.py will impact Sep's PR.

Awesome thanks. I've cleanup up all the unneeded commented code in the operations schema you pointed out and will wait until Sepehr's PR gets merged.

**Pull request merged**